test/e2e/basic_test.go

```diff
@@ -614,22 +642,83 @@ func getRoundNumber(data *TestData, podName string) (uint64, error) {
 	return 0, fmt.Errorf("did not find roundNum in OVSDB result")
 }
 
+func getAntreaPodName(t *testing.T, data *TestData, nodeName string) string {
+	antreaPodName, err := data.getAntreaPodOnNode(nodeName)
+	if err != nil {
+		t.Fatalf("Error when retrieving the name of the Antrea Pod running on Node '%s': %v", nodeName, err)
+	}
+	t.Logf("The Antrea Pod for Node '%s' is '%s'", nodeName, antreaPodName)
+	return antreaPodName
+}
+
+func restartAntreaAgentPod(t *testing.T, data *TestData, podName, nodeName string) {
```

Comment:
the name of this function is incorrect, it should be something like `restartAntreaAgentContainer`. We don't want to restart the `antrea-ovs` container for `testDeletePreviousRoundFlowsOnStartup`, as restarting OVS cleans up flows automatically.